Dockerfile

```diff
@@ -95,6 +91,15 @@ RUN \
     && echo 'Cleaning up installation files' >&2 && \
     apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
+# install required python packages from file
+COPY pydeps.txt /tmp/pydeps.txt
+COPY pypkgs.txt /tmp/pypkgs.txt
+RUN echo 'Installing python3 packages' >&2 \
+    && pip3 install -r /tmp/pydeps.txt \
```

Comment:
why two files (pydeps and pypkgs)?

Reply:
the pypkgs.txt contains all the actually wanted python packages like ASN1, i.e. these match the `apt-get install <python3-foo>`. The ones in `pydeps.txt` are prerequisites to be able to install those packages, i.e. they were not required when installing via `apt`.

Reply:
ah and that way I can easily remove these packages in pydeps.txt after installing the required packages ... I'll add this to the Dockerfile, might safe some bytes